src/types/onyx/Policy.ts

```diff
@@ -190,8 +190,8 @@ type QBOConnectionConfig = OnyxCommon.OnyxValueWithOfflineFeedback<{
     exportAccount: string;
     exportAccountPayable: string;
     accountPayable: string;
-    exportEntity?: ValueOf<typeof CONST.QUICKBOOKS_EXPORT_ENTITY>;
-    exportCompanyCard: ValueOf<typeof CONST.QUICKBOOKS_EXPORT_COMPANY_CARD>;
+    exportEntity?: ValueOf<typeof CONST.QUICKBOOKS_OUT_OF_POCKET_EXPENSE_ACCOUNT_TYPE>;
+    exportCompanyCard: ValueOf<typeof CONST.QUICKBOOKS_EXPORT_COMPANY_CARD_ACCOUNT_TYPE>;
```

Comment:
NAB: I don't think these keys `exportEntity` or `exportCompanyCard` really exist, shouldn't they be `reimbursableExpensesExportDestination` and `nonReimbursableExpensesExportDestination`?

<img width="684" alt="image" src="https://github.com/Expensify/App/assets/87341702/828ff656-1131-4fe4-b5f0-a31b2710e2ae">


This is causing the inputs to start empty because we read the data from non-existing properties

<img width="445" alt="image" src="https://github.com/Expensify/App/assets/87341702/d2edb30f-5224-444f-aabf-7f562414f01d">


I put this as NAB because this was already wrong and we can fix it in another PR if you prefer.

Reply:
Ugh, these bad setting names made it to the backend in Auth already because I didn't check if they were correct: https://github.com/Expensify/Auth/blob/3bebe7873f058bdc4f1df5c926f4a3d64f94171a/auth/lib/Policy.h#L111-L140

Reply:
These non-existing field values were added in [this PR](https://github.com/Expensify/App/pull/39191). I completely missed that during the code review.

I don't know why they have added additional fields to the TypeScript definition though. I already question that in [this PR](https://github.com/Expensify/App/pull/41079#discussion_r1583449067), but I should have realized that that also made it to the backend code

Reply:
ahhh, so that explains why they are empty

Reply:
Starting to work here: https://github.com/Expensify/App/pull/41463